src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,65 @@
+.watch {
+  position: relative;
+
+  display: flex;
+  box-sizing: border-box;
+  justify-content: center;
+  align-items: center;
+  height: $watch-size;
+  width: $watch-size;
+
+  background-color: #c2c6c8;
+  background-image: url(../images/strelki.svg);
+  background-repeat: no-repeat;
+  background-size: cover;
+
+  border: 3px solid #000;
+  border-radius: $border-radius;
+
+  &__circle {
+    position: relative;
+    width: $circle-size;
+    height: $circle-size;
+
+    background-color: #4b9aaa;
+    border-radius: $border-radius;
+    border: 1px solid #000;
+
+    z-index: 1;
+  }
+
+  &__minute {
```

Comment:
Good job! The only thing I would recommend to change – to put minute hand above the second hand, because minute hand is smaller and sometimes second hand covers it. Also it seems that your background image is a little bit turned and the small lines don't match with the hand tip. Maybe rotating it a bit will help :)